### DISCUSSION TITLE
Bunch does not appear in System Preferences Notification panel?

### DISCUSSION THREAD

**Discussion by provuejim** [ttscoff/bunch-docs#198] [Q&A] [answered]
When I installed Bunch on my M1 MacBook Pro running Big Sur 11.6, everything went smoothly except for one issue -- setting up notifications.  Here’s my Preferences window (mostly because I want to demonstrate that Bunch is in fact actually running).

![image](https://user-images.githubusercontent.com/26842788/152662432-21cc972d-2c33-42ef-8be7-55dec438effa.png)

The problem is that Bunch doesn’t show up in the Notifications panel of System Preferences, so I can’t enable notifications. (I had no problem with the Accessibility preferences, btw.) I expected Bunch to appear between Books and Calendar, but it isn't anywhere on the list.

<img width="480" alt="image" src="https://user-images.githubusercontent.com/26842788/152662438-2ea77d8d-913e-4c3e-9b99-f8eef12c1a5d.png">

Quitting and relaunching Bunch didn't help, and neither has rebooting the computer. Any suggestions? Except for notifications, Bunch seems to be running fine.

**Comment by ttscoff** [comment, marked as answer]
It’s possible Bunch needs to attempt to actually send a notification before it registers for them. Could you try adding a notify command to a Bunch and seeing if that triggers it?

Thanks,
Brett

**Reply by provuejim** [reply]
That seems to have worked, thanks!
